### PR TITLE
Implement e2e to validate module's install process

### PR DESCRIPTION
**Pull request by leonampd**
Creates a new context to test if the module is installed correctly. It tests if a given ecommerce admin can log in admin panel and checks if a module menu item is visible on sidebar menu

### CODE REVIEW

File: Test/Acceptance/AdminContext.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    /**
+     * @Then I should see pagar.me as an item on left menu
+     */
+    public function iShouldSeePagarMeAsAnItemOnLeftMenu()
+    {
+        $page = $this->getSession()->getPage();
+        
+        $this->getSession()->wait(
```
Comment (lflimeira) [inline Test/Acceptance/AdminContext.php]: We could implement a Spin method and stop to use the wait method =)

File: Test/Acceptance/AdminContext.php [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        $admin = new stdClass;
+        $admin->username = $this->getEnvFromName('MAGENTO_ADMIN_USERNAME');
+        $admin->password = $this->getEnvFromName('MAGENTO_ADMIN_PASSWORD');
+
+        return $admin;
+    }
+
+    public function getModuleMenuBarId()
+    {
+        return '#menu-mundipagg-top-level';
```
Comment (lflimeira) [inline Test/Acceptance/AdminContext.php:96]: There isn't another property that doesn't have mundipagg?

**Review by lflimeira — CHANGES_REQUESTED** [review]
I think we should implement CI before we approve.

File: Test/Acceptance/AdminContext.php (hunk 1 above)
Comment (leonampd) [reply Test/Acceptance/AdminContext.php]: In this case the spin already implemented, isn't it? I mean, the second paramenter is a condition that checks if the page has been loaded.

**Review by lflimeira — APPROVED** [review]

**Review by lflimeira — APPROVED** [review]

**Pull request merged**